code/modules/cargo/packs.dm

```diff
@@ -201,7 +201,7 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 				/obj/item/gun/energy/taser,
 				/obj/item/gun/energy/nt_svalinn,
 				/obj/item/gun/energy/nt_svalinn)
-	cost = 4500
+	cost = 4000
```

Comment:
I think you can drop this to like 500 or less

Reply:
Make it 500? You mean 2 svalinns costed 4000?